templates/cpu.json

```diff
@@ -0,0 +1,38 @@
+{
+    "variables": {
+        "aws_access_key": "",
+        "aws_secret_key": ""
+    },
+    "builders": [
+        {
+            "type": "amazon-ebs",
+            "access_key": "{{user `aws_access_key`}}",
+            "secret_key": "{{user `aws_secret_key`}}",
+            "region": "us-east-2",
+            "source_ami": "ami-07c1207a9d40bc3bd",
```

Comment:
what are your thoughts on using [`source_ami_filter`](https://www.packer.io/docs/builders/amazon-ebs.html#source_ami_filter) instead of `source_ami`? this way we always build from the most recently published Ubuntu AMI instead of version pinning to the one here.

Reply:
Good idea.